submissions/OlamijiBadmos/html-forms/contact.html

```diff
@@ -7,6 +7,7 @@
     <title>Dufuna-Fem | Contact Page</title>
 </head>
 <body>
+    <h4>The products in the store are  <a href="product.html">Product Page</a> </h4>
```

Comment:
Include the link to the contact page too. Also, add the navigation links on the product.html file